src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -25,7 +25,9 @@
 import static org.folio.utils.UserPermissionsUtil.verifyUserHasAssignPermission;
 import static org.folio.utils.UserPermissionsUtil.verifyUserHasManagePermission;
 import static org.folio.utils.UserPermissionsUtil.verifyUserHasFiscalYearUpdatePermission;
-
+import static org.folio.utils.UserPermissionsUtil.verifyUserHasFiscalYearUpdat;
```

Comment:
this code will not compile, check that code is compling before commit

Reply:
sure, I am sorry.